core-bundle/src/Command/UserPasswordCommand.php

```diff
@@ -111,7 +112,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $affected = $this->connection->update(
             'tl_user',
-            ['password' => $hash, 'locked' => 0, 'loginAttempts' => $config->get('loginAttempts')],
+            [
+                'password' => $hash,
+                'locked' => 0,
+                'loginAttempts' => 0,
+                'pwChange' => $input->getOption('require-change'),
```

Comment:
The `pwChange` column is defined as `char(1) NOT NULL default ''`. Did you make sure that we can pass a boolean value in MySQL strict mode? Maybe it should be `$input->getOption('require-change') ? '1', ''` instead?

Reply:
It worked for me even in strict mode but I think it's a good idea to be explicit here and not rely on casting/transforming. 

It's changed in 1dcf4798bc845c23c2b1999d73e8e69e96010cc1